pylossless/tests/test_rejection.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+import shutil
+
+import pylossless as ll
+from pylossless.tests.test_datasets import load_openneuro_bids
+
+
+def test_rejection_policy():
+    """testing the rejection policy."""
+    raw, config, bids_root = load_openneuro_bids()
+
+    config.save("test_config.yaml")
+    pipeline = ll.LosslessPipeline('test_config.yaml')
+    not_in_1020 = ['EXG1', 'EXG2', 'EXG3', 'EXG4',
+                   'EXG5', 'EXG6', 'EXG7', 'EXG8']
+    pipeline.raw = raw.pick('eeg',
+                            exclude=not_in_1020).load_data()
+    pipeline.run_with_raw(pipeline.raw)
+
+    rejection_config_path = Path("test_rejection_config.yaml")
+    rejection_config = ll.Config()
+    rejection_config["ch_flags_to_reject"] = ['ch_sd', 'low_r']
+    rejection_config.save(rejection_config_path)
+
+    raw = ll.RejectionPolicy(rejection_config_path).apply(pipeline)
+    rejection_config_path.unlink()
+
```

Comment:
The typo I pointed out in `rejection.py` caused this class to not behave as expected. We should add tests to check that the code behaves as expected and not just that it runs without raising errors. something like:

```
 if clean_ch_mode == None:
    assert_array_equal(flagged_chs, raw.info["bads"])
elif clean_ch_mode == "drop":
    not_in_ch_names = [ch for ch in flagged_chs if ch not in raw.ch_names]
    assert_array_equal(not_in_ch_names, flagged_chs)
elif clean_ch_mode == "interpolate":
    # interpolate_bads will drop ch_names from raw.info["bads"]
    # so to make sure that the channel was interpolated, lets check it
    assert_array_equal(flagged_chs, raw.info["bads"])
```